README.md

```diff
@@ -78,11 +78,11 @@ to enable the bot version environment variable (run this if the version is
 ## Testing
 
 There is a testing deploy set up at
-https://dashboard.heroku.com/apps/sympy-bot-testing. Currently it is
+https://dashboard.heroku.com/apps/sympy-bot-testing. Currently, it is
 configured to run on https://github.com/asmeurer/GitHub-Issues-Test.
 
 To test, push to a separate branch (`master` has branch protection) on this
-repo (you can also set up a separate testing deploy for your fork if you
+repo (you can also seta up separate testing deploy for your fork if you
```

Comment:
This line still needs to be addressed, as suggested previously.

Reply:
thanks, @sidhantnagpal for suggestion

Reply:
thanks, @sidhantnagpal for the suggestion